src/cloud/musescorecom/imusescorecomcloudservice.h

```diff
@@ -19,33 +19,36 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-#ifndef MU_CLOUD_ICLOUDPROJECTSSERVICE_H
-#define MU_CLOUD_ICLOUDPROJECTSSERVICE_H
+#ifndef MU_CLOUD_MUSESCORECOM_ICLOUDSERVICE_H
+#define MU_CLOUD_MUSESCORECOM_ICLOUDSERVICE_H
 
 #include <QUrl>
 
 #include "modularity/imoduleexport.h"
 #include "progress.h"
 
-#include "cloudtypes.h"
+#include "cloud/cloudtypes.h"
 
 class QIODevice;
 class QString;
 
 namespace mu::cloud {
-class ICloudProjectsService : MODULE_EXPORT_INTERFACE
+class IMuseScoreComCloudService : MODULE_EXPORT_INTERFACE
 {
-    INTERFACE_ID(ICloudProjectsService)
+    INTERFACE_ID(IMuseScoreComCloudService)
 
 public:
-    virtual ~ICloudProjectsService() = default;
+    virtual ~IMuseScoreComCloudService() = default;
 
-    virtual framework::ProgressPtr uploadScore(QIODevice& scoreData, const QString& title, Visibility visibility = Visibility::Private,
+    virtual QUrl scoreManagerUrl() const = 0;
```

Comment:
I would suggest keeping this method in ICloudConfiguration

Reply:
Let's leave it as it is now. I like this method better in service